src/main/scala/nl.knaw.dans.easy.solr4files/components/Bag.scala

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (C) 2017 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.solr4files.components
+
+import java.net.URL
+
+import nl.knaw.dans.easy.solr4files.{ FileToShaMap, SolrLiterals }
+import nl.knaw.dans.lib.logging.DebugEnhancedLogging
+
+import scala.util.Try
+import scala.xml.Elem
+
+case class Bag(storeName: String,
+               bagId: String,
+               private val vaultIO: VaultIO
+              ) extends DebugEnhancedLogging {
+
+  private def getDepositor: String = Try {
+    val key = "EASY-User-Account"
+    vaultIO.linesFrom(storeName, bagId, "bag-info.txt")
+      .filter(_.trim.startsWith(key))
+      .map(_.trim.replace(key, "").trim.replace(":", "").trim)
```

Comment:
is all this trimming really necessary?
I'd say you do a `.map(_.trim)` before the `filter` line: that way you don't need to do this `trim` twice.
Then using `collectFirst` you can do this easier than with this `map(...).head` construct.

Reply:
Or can I load this file like a `.properties` file? The ':' in stead of '=' pushed me in the DIY solution.